resources/options_lookup.tsv

```diff
@@ -1316,6 +1316,7 @@ Cooling Setpoint	76F w/ Building America season	ResStockArguments	hvac_control_c
 Cooling Setpoint	77F	ResStockArguments	hvac_control_cooling_weekday_setpoint_temp=77	hvac_control_cooling_weekend_setpoint_temp=77	use_auto_cooling_season=false	hvac_control_cooling_season_period=auto
 Cooling Setpoint	78F	ResStockArguments	hvac_control_cooling_weekday_setpoint_temp=78	hvac_control_cooling_weekend_setpoint_temp=78	use_auto_cooling_season=false	hvac_control_cooling_season_period=auto
 Cooling Setpoint	80F	ResStockArguments	hvac_control_cooling_weekday_setpoint_temp=80	hvac_control_cooling_weekend_setpoint_temp=80	use_auto_cooling_season=false	hvac_control_cooling_season_period=auto
+Cooling Setpoint	Void
```

Comment:
These options are in the TSVs but were not in `options_lookup`. The integrity checks did not catch this as the "Void" options are never sampled (and are never meant to be sampled). Because these options are in the `options_saturation.csv` they are not found in `options_lookup` and the table doesn't get formatted correctly and throws an error.